server/sonar-server/src/main/java/org/sonar/server/permission/ws/PermissionsWsParametersBuilder.java

```diff
@@ -78,8 +78,8 @@ public static void createGroupNameParameter(NewAction action) {
       .setExampleValue("sonar-administrators");
   }
 
-  public static void createOrganizationParameter(NewAction action) {
-    action.createParam(PARAM_ORGANIZATION)
+  public static NewParam createOrganizationParameter(NewAction action) {
+    return action.createParam(PARAM_ORGANIZATION)
       .setDescription("Key of organization, used when group name is set")
       .setExampleValue("my-org")
       .setSince("6.2")
```

Comment:
this line should not be in the builder method IMO because it's very context sensitive